WordPress/src/androidTest/java/org/wordpress/android/WordPressTest.kt

```diff
@@ -2,9 +2,9 @@ package org.wordpress.android
 
 import org.wordpress.android.modules.DaggerAppComponentTest
 
-class WordPressTest : WordPress() {
+class WordPressTest : WordPressApp() {
```

Comment:
For now, `WordPressTest` is extended from `WordPressApp`. When Hilt added, `WordPressTest` will extend `WordPress` and inject and init `AppInitializer` differently from `WordPressApp`.

Reply:
Sounds good.  I've been able to build and run without error